### ISSUE TITLE
Skrivekunst vs Skriving

### ISSUE THREAD

**Issue by bibliomari** [realfagstermer/realfagstermer#412]
Kan slås sammen, ser det ut til. Skrivekunst BF Skriving. Protester?

**Comment by BioHeidi** [comment]
Nei, ingen protest herfra.

**Comment by Hypsibius** [comment]
Ja. Protest innlagd! Skriving omfattar både 'kreativ skriving' og akademisk skriving, som hos oss heiter 'faglig forfatterskap'. Trur det er bedre å spesifisere, sjølv om det evt kan medføre litt manuell rydding.  

Mitt forslag: 

'Skriving' erstattast avfølgande termar:

'Faglig forfatterskap' (altså akademisk skriving)

'Kreativ skriving', legg til EN 'Creative writing'

'Teknisk skriving'. Bør denne samordnast med 'Journalføring' og 'Rapportskriving'?

Og så framstår det som - uhm - underleg - at det heiter kreativ skriving, rapportskriving med rettskrivNing! Men så eer jo eg ein barka nynorsking....

'Maskinskriving' kan vel stå, men med tillegget EN 'typing'

Og masse SO!
